addons/website_profile/static/src/js/website_profile.js

```diff
@@ -116,6 +116,19 @@ publicWidget.registry.websiteProfileEditor = publicWidget.Widget.extend({
     },
 });
 
+publicWidget.registry.websiteProfileNextRankCard = publicWidget.Widget.extend({
+    selector: '.o_wprofile_progress_circle',
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.$('g[data-bs-toggle="tooltip"]').tooltip();
```

Comment:
Well the doc says:

> Tooltips are opt-in for performance reasons, so you must initialize them yourself.

So you are right, we probably have to have this code.
A bit sad to implement a full publicWidget for a tooltip, but oh well ^^